### ISSUE TITLE
Add local WS numbers beside each WS in the tree view

### ISSUE THREAD

**Issue by Mati-Roy** [oughtinc/mosaic#708]
This would make easier to refer to them in the Malicious Team Google Docs.

Discussion reference with Meghan and Matthias: https://app.slack.com/client/T3L8K3NMT/CL6PM7L94/thread/CL6PM7L94-1566937470.082300?thread_ts=1564795608.023600

**Comment by brachbach** [comment]
I don't think we should call that the global number, I think we should call it the tree number

so then we'd have "workspace number" and "tree number"

**Comment by Mati-Roy** [comment]
oh right, I agree
